.github/workflows/merge-check.yml

```diff
@@ -21,27 +21,8 @@ jobs:
       - name: Checkout Repo
         uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3
 
-      - name: Cache Gradle Files
-        uses: actions/cache@e12d46a63a90f2fae62d114769bbf2a179198b5c # v3
-        with:
-          path: |
-            ~/.gradle/caches/
-            ~/.gradle/wrapper/
-          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
-          restore-keys: |
-            ${{ runner.os }}-gradle-
-
-      - name: Cache CVE database Files
-        uses: actions/cache@e12d46a63a90f2fae62d114769bbf2a179198b5c # v3
-        with:
-          path: |
-            ~/.gradle/dependency-check-data/
-          # Use the hash of the gradle files as the key for now,
-          # which contains the `org.owasp.dependencycheck` plugin version.
-          # The database might depend on other things as well, like date, but this is good enough.
-          key: dependency-check-${{ hashFiles('build.gradle.kts') }}
-          restore-keys: |
-            dependency-check-
```

Comment:
Hmm, wait, I think we need still this, because this is not a Gradle controlled folder! 

https://github.com/gradle/gradle-build-action?tab=readme-ov-file#which-content-is-cached

Reply:
I'm going to wait for this decision first: https://github.com/detekt/sarif4k/pull/86#issuecomment-1921842131 then I'll go back here.